_data/companies.yml

```diff
@@ -306,13 +306,6 @@
   events: Restricted
   last_update: 2020-03-12
 
-- name: CERN [[1]](https://hse.cern/news-article/coronavirus-information-measures-and-recommendations)
-  wfh: Planned
-  travel: Restricted
-  visitors: Restricted
-  events: Restricted
-  last_update: 2020-03-10
-
```

Comment:
It looks like this PR is trying to overwrite some recent changes. Would you mind undoing this?